Keas.Mvc/ClientApp/components/Keys/CreateKey.tsx

```diff
@@ -11,6 +11,7 @@ interface IProps {
     onOpenModal: () => void;
     closeModal: () => void;
     searchableTags: string[];
+    keyCodes: string[];
```

Comment:
I kinda think I'd prefer instead of passing along the entire list of keycodes, we instead pass a function which can be called to determine if a keycode already exists.  unless there is another reason createKey should know the full list of codes